src/blog/2024/05/node-red-mind-stack-with-flowfuse.md

```diff
@@ -0,0 +1,69 @@
+
+<!--more-->
+
+## FlowFuse: Integrated IoT Deployment
+
+With FlowFuse we have bundled up 3 of those 4.  First and foremost, the FlowFuse platform deploys Node-RED in two main ways.  A cloud based deployment and remote deployment with management from the FlowFuse platform.  Part of this management piece leverage MQTT between the devices for management.  Because of this we have a built in MQTT broker that allows for communication between Node-RED instance within the FlowFuse platform.  Lastly, with the release of Dashboards 2.0 from FlowFuse we have introduced the next generation of visualizations for the Node-RED platform.
+
+
+## Optimizing IoT with the MIND Stack
+
+It often makes sense to deploy a full MING stack, but in some deployments, it might be necessary to deploy a more simplistic version of the MING stack.  This is where MIND comes into play.
+
+
+### FlowFuse MQTT: Simplified Communications
+
+FlowFuse has a built-in feature called projects, which leverages MQTT, that allows the communication of data between FlowFuse runtimes of Node-RED.  One caveat is that this MQTT broker is only available within the FlowFuse platform. What this means is there needs to be some form of translation to be done within Node-RED.  This isn’t a big deal for small deployments, because often Node-RED runtimes at the edger are collecting data from various sources that aren’t MQTT.  The flow of data is as follows: 
+
+Sensor > Node-RED(FlowFuse Device Agent) > MQTT Encapsulated by FlowFuse > Node-RED(FlowFuse Platform) > InfluxDB
+
+!["Screenshot showing the flow of data: Sensor > Node-RED(FlowFuse Device Agent) > MQTT Encapsulated by FlowFuse > Node-RED(FlowFuse Platform) > InfluxDB"](./images/sensor-data-mqtt-node-red-dashboard-influxdb.png ""){data-zoomable}
+
+A key benefit of having the MQTT broker encapsulated by FlowFuse is that it becomes extremely easy to set up.  With the project nodes, there isn’t a need to configure certificates or security credentials because it is only exposed internally to the FlowFuse platform.
+
+
+### InfluxDB: Efficient Data Storage
+
+InfluxDB doesn’t change much in the MIND configuration compared to how it is deployed in the MING stack, minus the fact that the data will be stored next to the FlowFuse Platform instead of the cloud.  
+
+
+### Data Visualization with Dashboard 2.0
+
+FlowFuse's Dashboard 2.0 makes it incredibly easy to visualize your data.  You can create beautiful and informative dashboards in minutes. Simply drag and drop your desired widgets onto the canvas, and configure them with a few clicks.  
+
+We have gone through the step of providing you with an example dashboard for visualizing your MIND stack.   
+
+
+_Sample Dashboard_
```

Comment:
?

Reply:
Updated with renderFlow.  Thanks for catching this.